### DISCUSSION TITLE
Redux Toolkit is UI-agnostic, what does this mean exactly?

### DISCUSSION THREAD

**Discussion by anthonytranDev** [reduxjs/redux#4720] [General]
### Context

I'm currently reading through the documentation - https://redux-toolkit.js.org/rtk-query/overview

I wanted to quote a specific bullet made here, as I don't understand it much.
 
_"Redux Toolkit is UI-agnostic, RTK Query's functionality can be used with any UI layer"_

### Presumptions

Being UI agnostic means to me

- Components can be rendered client and server
- Used alongside `react-native`

**Comment by markerikson** [comment]
It means that it is not directly tied to any specific UI library.

We assume that _most_ Redux users are using it with React.  But, you can use it with _anything_: Vue, Angular, Svelte, Ember, jQuery, vanilla JS, or no UI layer at all.
